packages/react/src/header/HeaderBrand.tsx

```diff
@@ -74,40 +92,57 @@ export function HeaderBrand({
 				>
 					{secondLogo}
 				</Flex>
-
-				{dividerPosition === 'after' && <DividingLine />}
 			</Flex>
 
-			<Stack
-				as={Link}
-				color="text"
-				css={{
-					textDecoration: 'none',
-					':hover': packs.underline,
-				}}
-				focusRingFor="keyboard"
-				href={href}
-				justifyContent="center"
+			<Flex
+				gap={GAP_REM}
+				css={
+					dividerPosition === 'after'
+						? {
+								[`${tokens.mediaQuery.min.lg}`]: {
+									marginLeft: `calc(-${
+										hasRightContent ? mapSpacing(GAP_REM) : 0 // Offset the gap when there is right content
+									} - 1px)`, // Hide the divider when the heading text flows to the second row
```

Comment:
`-1px` should be `-${tokens.borderWidth.sm}px`

Reply:
Ahh good point!

- [ ] -${tokens.borderWidth.sm}px